wallet/src/background/Permissions.ts

```diff
@@ -224,6 +224,16 @@ class Permissions {
         );
     }
 
+    public async clear(origin: string) {
```

Comment:
Clear kind of implies removing all, maybe call this delete?

Reply:
Done